changelogs/fragments/bump_215.yaml

```diff
@@ -0,0 +1,6 @@
+---
+release_summary: >
```

Comment:
we dont need release_summary here, it would be required in the maor release PR thats why changelog CI action is failing